pkg/docker-compose/build.go

```diff
@@ -44,8 +44,9 @@ func (m *Mixin) Build(ctx context.Context) error {
 	}
 
 	dockerfileLines := fmt.Sprintf(`RUN apt-get update && apt-get install -y curl && \
-curl -fL "https://github.com/docker/compose/releases/download/v%s/docker-compose-linux-x86_64" -o /usr/local/bin/docker-compose && \
-chmod +x /usr/local/bin/docker-compose`, dockerComposeVersion)
+    curl -fL "https://github.com/docker/compose/releases/download/v%s/docker-compose-linux-x86_64" -o /usr/local/lib/docker/cli-plugins/docker-compose --create-dirs && \
+    chmod +x /usr/local/lib/docker/cli-plugins/docker-compose && \
```

Comment:
Why `ln` over setting it in `-o`, just generally curious

Reply:
This has changed slightly after merging the latest from `main` and fixing the conflicts. Initially, I updated it to use `ADD` to download the file and then `RUN ln -s` to create the symlink (https://github.com/getporter/docker-compose-mixin/pull/48/commits/63bfacc0fcceacbe0cb7974b8e8d2b46ee4f5555). However, after doing that, it occurred to me that instead of creating a symlink in /usr/local/bin, we could just update the PATH:

    ADD --chmod=755 https://github.com/docker/compose/releases/download/v%s/docker-compose-linux-x86_64 /usr/local/lib/docker/cli-plugins/docker-compose
    ENV PATH="$PATH:/usr/local/lib/docker/cli-plugins"

This seems like a nicer way of ensuring that `docker-compose` can still be used. I've tested and pushed that change up, but could revert to using a symlink or just installing the binary into both folders, whichever you think is cleaner.

Thanks for the review!